### PR TITLE
Fix nf-core sync --all

### PR DESCRIPTION
**Pull request by ewels**
First attempt at fixing the automatic synchronisation for all pipelines: https://github.com/nf-core/tools/issues/503

Basically just stripped out the code to try to figure out authentication automatically, instead just pass this over the command line. Should make it easier to run this code manually and easier to debug.

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] Documentation in `docs` is updated
 - [ ] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

### CODE REVIEW

**Comment by ewels** [comment]
Example pull-request generated by this code: https://github.com/nf-core/ampliseq/pull/132

Might be nice to add in a bit more help text to the PRs telling people how to deal with them... x-ref https://github.com/nf-core/tools/issues/509

**Comment by ewels** [comment]
Ok, instead of putting the docs in the comment I have updated it to link to https://nf-co.re/developers/sync](https://nf-co.re/developers/sync

I will make a separate PR there with improved documentation.

This PR is now ready for review and merge!

File: README.md [hunk 1]
```diff
@@ -534,8 +534,7 @@ with `--username` and `--repository`.
 
 To create the pull request, a personal access token is required for API authentication.
 These can be created at [https://github.com/settings/tokens](https://github.com/settings/tokens).
```
Comment (drpatelh) [inline README.md]: How exactly do you do this? When I click on that link and try and "Generate new token" there are loads of options that you can select?!

Comment (ewels) [reply README.md]: Ah, you mean the different scopes? This is just to limit what can be done with the auth token for security. The simplest way is just to click all of the tick boxes ✅✅✅

I think that only `repo` is required though, and probably not even all of those.

**Pull request merged**